dev/TabView/InteractionTests/TabViewTests.cs

```diff
@@ -572,7 +572,7 @@ public void GamePadTest()
 
                 GamepadHelper.PressButton(null, GamepadButton.LeftThumbstickUp);
                 Wait.ForIdle();
-                Verify.IsTrue(toggleThemeButton.HasKeyboardFocus, "GamePad Up should move to toggle theme button");
+                Verify.IsTrue(toggleInnerFrameDimensions.HasKeyboardFocus, "GamePad Up should move to toggle inner frame dimensions button");
```

Comment:
Should we add something to the test page here to receive focus? It is weird to have the test depend on the surrounding test chrome. The toggleThemeButton/toggleInnerFrameDimensions doesn't really matter here, it is just what happens to be above the tab control - right?

Reply:
Probably although the change in this PR isn't functional, I simply changed the local variable name.

Reply:
Yeah, it's fine to not fix the test, since it not really related to your change.